spec/data-model/README.md

```diff
@@ -1,15 +1,34 @@
 # DRAFT MessageFormat 2.0 Data Model
 
-To work with messages defined in other syntaxes than that of MessageFormat 2,
-an equivalent data model representation is also defined.
-Implementations MAY provide interfaces which allow
-for MessageFormat 2 syntax to be parsed into this representation,
-for this representation to be serialized into MessageFormat 2 syntax
-or any other syntax,
-for messages presented in this representation to be formatted,
-or for other operations to be performed on or with messages in this representation.
+This section defines a data model representation of MessageFormat 2 _messages_.
 
 Implementations are not required to use this data model for their internal representation of messages.
+Neither are they required to provide an interface that accepts or produces
+representations of this data model.
+
+The reason this specification provides a data model is to allow interchange of 
+the logical representation of a _message_ between different implementations.
+This includes mapping legacy formatting syntaxes (such as MessageFormat 1)
+to a MessageFormat 2 implementation.
+Another use would be in converting to or from translation formats without 
+the need to continually parse and serialize all or part of a message.
+
+Implementations that expose APIs supporting the production or consumption of a
+_message_ as a data structure are encouraged to use this data model.
+
+This data model provides these capabilities:
+- any MessageFormat 2 message (including future versions)
+  can be parsed into this representation
+- this data model representation can be serialized as a well-formed
+  MessageFormat 2 message
```

Comment:
Needs to include a round tripping constraint, something like (but not these words).

If I take an MF2 message M, convert to the data model D, then convert back to an MF2 message M', then M and M' will be functionally equivalent (eg produce the same results with the same input)

Reply:
```suggestion
  MessageFormat 2 message
- parsing a MessageFormat 2 message into a data model representation
  and then serializing it results in an equivalent message
```

Reply:
Note that the resulting message can vary in terms of whitespace normalization, variant ordering, and such

```suggestion
MessageFormat 2 message
- parsing a MessageFormat 2 message into a data model representation
  and then serializing it results in an equivalently functional message
```